### PR TITLE
Revert "Make gas fit in 2 limbs (#1261)"

### PR DESCRIPTION
**Pull request by Nashtare**
This reverts commit 0f19cd0dbc25f9f1aa8fc325ae4dd1b95ca933b3.

This change was added for testing purposes only, but isn't intended to be kept in the long term. Now that all RC1 items are merged, this can be reverted.

### CODE REVIEW

File: evm/src/cpu/syscalls_exceptions.rs [hunk 1]
```diff
@@ -101,18 +101,17 @@ pub(crate) fn eval_packed<P: PackedField>(
...
     // New top of the stack: current PC + 1 (limb 0), kernel flag (limb 1), gas counter (limbs 6 and 7).
     yield_constr.constraint(filter_syscall * (output[0] - (lv.program_counter + P::ONES)));
     yield_constr.constraint(filter_exception * (output[0] - lv.program_counter));
     // Check the kernel mode, for syscalls only
     yield_constr.constraint(filter_syscall * (output[1] - lv.is_kernel_mode));
-    // TODO: Range check `output[6] and output[7]`.
-    yield_constr.constraint(total_filter * (output[6] - lv.gas[0]));
-    yield_constr.constraint(total_filter * (output[7] - lv.gas[1]));
+    yield_constr.constraint(total_filter * (output[6] - lv.gas));
+    // TODO: Range check `output[6]`.
```
Comment (hratoanina) [inline evm/src/cpu/syscalls_exceptions.rs:113]: Done in a previous PR.

File: evm/src/cpu/syscalls_exceptions.rs [hunk 2]
```diff
@@ -297,15 +294,15 @@ pub(crate) fn eval_ext_circuit<F: RichField + Extendable<D>, const D: usize>(
...
         yield_constr.constraint(builder, constr);
     }
-    // TODO: Range check `output[6]` and `output[7].
     {
-        let diff = builder.sub_extension(output[6], lv.gas[0]);
+        let diff = builder.sub_extension(output[6], lv.gas);
         let constr = builder.mul_extension(total_filter, diff);
         yield_constr.constraint(builder, constr);
     }
+    // TODO: Range check `output[6]`.
```
Comment (hratoanina) [inline evm/src/cpu/syscalls_exceptions.rs:302]: Same.

**Comment by Nashtare** [comment]
@wborgeaud Are you good to revert this now?

**Review by wborgeaud — APPROVED** [review]
Yes all good 👍

**Pull request merged**